src/xcode/ENA/ENA/Source/Services/CCLService/CCLScenariosHelper.swift

```diff
@@ -0,0 +1,53 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+class CCLScenariosHelper {
+	
+	// MARK: - Init
+			
+	init(cclService: CCLServable, store: HealthCertificateStoring) {
+		self.cclService = cclService
+		self.store = store
+	}
+	
+	// MARK: - Private
+
+	private let cclService: CCLServable
+	private let store: HealthCertificateStoring
+
+	// MARK: - Internal
+
+	func showAdmissionScenarios() -> Result<SelectValueViewModel, DCCAdmissionCheckScenariosAccessError> {
```

Comment:
Still think the naming is off, new suggestion:
```suggestion
	func viewModelForAdmissionScenarios() -> Result<SelectValueViewModel, DCCAdmissionCheckScenariosAccessError> {
```